diff --git a/qa/tests/eeschema/sch_io/geda/test_geda_sch_import.cpp b/qa/tests/eeschema/sch_io/geda/test_geda_sch_import.cpp
--- a/qa/tests/eeschema/sch_io/geda/test_geda_sch_import.cpp
+++ b/qa/tests/eeschema/sch_io/geda/test_geda_sch_import.cpp
@@ -2069,7 +2069,7 @@
     std::unique_ptr<LIB_SYMBOL> sym;

     // Use a known .sym file from test data instead
-    wxString symPath = KI_TEST::GetEeschemaTestDataDir() + wxT( "/io/geda/7400-1.sym" );
+    std::string symPath = KI_TEST::GetEeschemaTestDataDir() + "/io/geda/7400-1.sym";
     wxTextFile file;

     BOOST_REQUIRE( file.Open( symPath ) );
